src/modules/Tables.js

```diff
@@ -228,14 +231,16 @@ export function LootHistoryTable(props) {
   var itemLookup = {};
   var tierLookup = {};
   for (const item of props.items) {
-    itemLookup[item.id] = item.name;
+    itemLookup[item.id] = <a href={item.url}>{item.name}</a>
     tierLookup[item.id] = item.tier;
   }
 
   const raidDaySort = (a, b) => {
     const raidDayA = props.raidDays.find(x => x.id === a.raid_day_id);
     const raidDayB = props.raidDays.find(x => x.id === b.raid_day_id);
-    return Date.parse(raidDayA.date) - Date.parse(raidDayB.date);
+    if (raidDayA && raidDayB) {
```

Comment:
Were you getting errors here?  What happens when the sort function here returns `null` for everything?